infrastructure/json/src/main/java/tech/pegasys/teku/infrastructure/json/types/StringBasedPrimitiveTypeDefinition.java

```diff
@@ -105,6 +113,12 @@ public void serializeOpenApiType(final JsonGenerator gen) throws IOException {
     if (format.isPresent()) {
       gen.writeStringField("format", format.get());
     }
+    if (minLength.isPresent()) {
+      gen.writeNumberField("minLength", minLength.get());
+    }
+    if (maxLength.isPresent()) {
+      gen.writeNumberField("maxLength", maxLength.get());
+    }
```

Comment:
what should be the effect here in the swagger definitions?

Reply:
If you define this fields in a string they will be used to limit the input allowed in the swagger interface.

Reply:
yep just our swagger output is hand-done so if it was first use it would have needed enhancing...